### DISCUSSION TITLE
dev services (vault) log information disappear from console

### DISCUSSION THREAD

**Discussion by fredofm** [quarkusio/quarkus#29273] [Q&A] [answered]
Hi all,

I'm trying to use "io.quarkiverse.vault:**quarkus-vault:**2.1.0" with dev services in dev mode (quarkus:dev).

The vault docker image is started automatically as expected and obviously the server has no secrets inside.

So I decided to use the property "**quarkus.vault.devservices.init-commands**: 'vault kv put secret/config password="p@$$w0rd"" to populate secrets at the startup. 

And it seems that the command is not ok for quarkus (I tried the command in the started docker container and it works).

The problem is that the warning message with the command information is only shown for a millisecond and then it's collapsed / overwritten with the next console information.

I was looking the way to log that information to a file or configure the console to avoid overwriting the information but I didn't find any clue on how to do it.

Questions;

- What is the right way to populate your vault when using dev services?.
- Is there any way to avoid overwriting the console information?

[a comment by an automated account is not shown]

**Comment by gsmet** [comment]
I recommend you to ask your question here instead: https://github.com/quarkiverse/quarkus-vault/discussions as it will ping the other people working on the extension. Providing a reproducer there might also be helpful.

**Reply by fredofm** [reply]
Thank you for the advice. I will do that :)

**Reply by fredofm** [reply]
I posted the same question here https://github.com/quarkiverse/quarkus-vault/discussions/93 and I have added more information.

**Comment by vsevel** [comment]
Any Idea @kdubb ?

**Comment by kdubb** [comment, marked as answer]
I answered in the vault discussion.

The original issue is because `vault` is in the arguments (when these are the arguments to the `vault` cli).

The logging issue is just how the rollup works because regardless of the initializers being run the service is still started. Unless there is a way for warnings/errors to always be displayed even after rollup.

**Reply by sberyozkin** [reply]
Perhaps something similar to #28533 can be done for Vault as well

**Reply by fredofm** [reply]
@kdubb Thanks a lot for your tips. I'will check it.
@sberyozkin It would be nice.
